src/lib/Core.jsx

```diff
@@ -122,12 +126,24 @@ function Core({
     return answers.map((answer, index) => {
       if (answerSelectionType === 'single') {
         // correctAnswer - is string
-        answerBtnCorrectClassName = (`${index + 1}` === correctAnswer ? 'correct' : '');
-        answerBtnIncorrectClassName = (`${userInputIndex}` !== correctAnswer && `${index + 1}` === `${userInputIndex}` ? 'incorrect' : '');
+        answerBtnCorrectClassName = `${index + 1}` === correctAnswer ? 'correct' : '';
+        answerBtnIncorrectClassName = `${userInputIndex}` !== correctAnswer
+        && `${index + 1}` === `${userInputIndex}` ? 'incorrect' : '';
+
+        if (userInputIndex === undefined && `${index + 1}` !== correctAnswer) {
+          answerBtnIncorrectClassName = ' unanswered';
+        }
       } else {
         // correctAnswer - is array of numbers
-        answerBtnCorrectClassName = (correctAnswer.includes(index + 1) ? 'correct' : '');
-        answerBtnIncorrectClassName = (!correctAnswer.includes(index + 1) && userInputIndex.includes(index + 1) ? 'incorrect' : '');
+        answerBtnCorrectClassName = correctAnswer.includes(index + 1)
+          ? 'correct'
+          : '';
+        answerBtnIncorrectClassName = !correctAnswer.includes(index + 1)
+        && userInputIndex?.includes(index + 1) ? 'incorrect' : '';
+
+        if (userInputIndex === undefined && !correctAnswer.includes(index + 1)) {
+          answerBtnIncorrectClassName = ' unanswered';
```

Comment:
there is a space at the beginning. is it intended?

Reply:
Thanks for the catch